.github/workflows/aws_tests.yaml

```diff
@@ -22,7 +22,9 @@ jobs:
 
       - name: Install python packages & dependencies
         run: |
-          pip install runhouse[aws] pytest tqdm datasets dask
+          pip install runhouse[aws]
+          pip install -r tests/requirements.txt
+          pip install -r tests/test_requirements/aws_test_requirements.txt
```

Comment:
The aws_test_reqs.txt installs additional packages that are required when executing tests related to AWS. Other tests (such as test_cluster.yaml), require only runhouse[aws], and do not require the packages listed in the aws_test_reqs.txt. To avoid unnecessary installations, I separated the runhouse[aws] install from the aws reqs file.